packages/cashier/src/types/stores/client-store.types.ts

```diff
@@ -24,4 +24,16 @@ export type TClientStore = {
     verification_code: {
         payment_withdraw: string;
     };
+    setVerificationCode: (code: string, action: string) => void;
+    updateAccountStatus: () => Promise<void>;
+    mt5_login_list: {
```

Comment:
For this prop, there is a type in the `@deriv/api-types`. You can check my [PR](https://github.com/binary-com/deriv-app/pull/6305/files#diff-0441c8476986d7c4588784c191e6b37bbc4f3ff52f694df9d0539598ec96be5eR19) for this.

### TLDR; 
`mt5_login_list: Array<DetailsOfEachMT5Loginid>;`

Reply:
@hamid-deriv Thank you 🌹